docker-compose.yml

```diff
@@ -102,7 +102,6 @@ services:
       - POSTGRES_DB=postgis
       - PGUSER=username
       - PGPASSWORD=password
-      - PGHOST=localhost
```

Comment:
https://github.com/stac-utils/pgstac/issues/166#issuecomment-1446525911

Reply:
still getting 

```
Run pypgstac pgready --dsn ***0.0.0.0:5432/postgis
[13](https://github.com/NASA-IMPACT/veda-backend/actions/runs/4383620233/jobs/7674104761#step:7:14)
error connecting in 'pool-1': connection failed: server closed the connection unexpectedly
[14](https://github.com/NASA-IMPACT/veda-backend/actions/runs/4383620233/jobs/7674104761#step:7:15)
	This probably means the server terminated abnormally
[15](https://github.com/NASA-IMPACT/veda-backend/actions/runs/4383620233/jobs/7674104761#step:7:16)
	before or while processing the request.
```

cc @bitner

Reply:
note: CI won't necessary `fail` if ☝️ step fails so we should add a `test` to make sure this step worked

Reply:
fixed: pgstac needed a bit more time to get up 🥳